From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 15:02:10 -08:00
From: khali@linux-fr.org
Subject: [PATCH] I2C: Make i2c list terminators explicitely unsigned

Shouldn't the i2c list terminators be explicitely declared as unsigned?
I'd hope it to help code analysis tools and possibly avoid false
positives. Coverity's SWAT pointed my attention to these constants.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/include/linux/i2c.h b/include/linux/i2c.h
--- a/include/linux/i2c.h	2005-03-02 16:32:14 -08:00
+++ b/include/linux/i2c.h	2005-03-02 16:32:14 -08:00
@@ -299,8 +299,8 @@
 };
 
 /* Internal numbers to terminate lists */
-#define I2C_CLIENT_END		0xfffe
-#define I2C_CLIENT_ISA_END	0xfffefffe
+#define I2C_CLIENT_END		0xfffeU
+#define I2C_CLIENT_ISA_END	0xfffefffeU
 
 /* The numbers to use to set I2C bus address */
 #define ANY_I2C_BUS		0xffff
